### DISCUSSION TITLE
llama2.mojo:519:15: error: cannot pass 'fn[Int](j = Int) capturing -> None' value, parameter expected 'Int' vectorize[_acc, nelts](size)

### DISCUSSION THREAD

**Discussion by agbarbosa** [tairov/llama2.mojo#83] [General]
Hello team,

I am trying to run the example mojo command line (Ubuntu 20.04.6): 
```bash
mojo llama2.mojo stories15M.bin -s 100 -n 256 -t 0.5 -i "Mojo is a language"
``` 

and getting the error: 
```
llama2.mojo:519:15: error: cannot pass 'fn[Int](j = Int) capturing -> None' value, parameter expected 'Int' vectorize[_acc, nelts](size)
```

```bash
$ mojo --version
mojo 0.7.0 (af002202)
```

Could someone please help and check if there's something wrong in the source code? I am new to the language and starting to learn it.

**Comment by mikowals** [comment]
The latest Mojo version (24.1.0) flipped the arguments of `vectorize` around from what they were in 0.7.0.  Since the current `master` branch is compatible with the latest version you will need to either upgrade your Mojo version or use an older commit.

**Reply by agbarbosa** [reply]
@mikowals - updated Mojo to version 24.1.0 by using the instructions here: https://docs.modular.com/engine/get-started - THAT WORKED!  Thanks again for your prompt response and being precise in the solution!

**Comment by agbarbosa** [comment]
Oh... I thought I was using the latest Mojo version. Thanks @mikowals for the reply! I will work on the mojo update on my environment.
